packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -756,9 +761,24 @@ class TextSelectionGestureDetector extends StatefulWidget {
   /// time (within [kDoubleTapTimeout]) to a previous short tap.
   final GestureTapDownCallback onDoubleTapDown;
 
+  /// Called after [onDoubleTapDown] when the pointer is lifted, unless
+  /// [onDoubleLongTapStart] is triggered.
+  final GestureTapUpCallback onDoubleTapUp;
+
   /// Called when a mouse starts dragging to select text.
   final GestureDragStartCallback onDragSelectionStart;
 
+  /// Called after [onDoubleTapDown], when the second tap is sustained for longer
```

Comment:
Does any platform do anything special here? Where double tap drag and a double tap wait drag does something different?

Reply:
Yes, iOS seems to highlight the word the touch landed on not immediately after the touch down event happens, but after a short interval.